invictum-test-core/src/main/java/com/github/invictum/pages/AbstractPage.java

```diff
@@ -111,6 +111,7 @@ protected String locatorValue(final String locatorKey) {
      * @return locator By
      */
     protected By locator(final String locatorKey, final String... parameters) {
+        addJQuerySupport();
```

Comment:
As for me it's bad practice to add JQuerry for all pages, even if user didn't use JQuerry at all.